src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -273,6 +273,28 @@ impl Default for CpuFeaturesTemplate {
     }
 }
 
+/// Error associated with creating CPU template from string.
+#[derive(Debug, PartialEq, Eq)]
+pub struct CpuFeaturesTemplateFromStringError;
+impl std::fmt::Display for CpuFeaturesTemplateFromStringError {
```

Comment:
Nit: This is a little more specific than it needs to be.
```suggestion
impl fmt::Display for CpuFeaturesTemplateFromStringError {
```